runbotics/runbotics-orchestrator-ui/src/src-app/Actions/jira-cloud.actions.ts

```diff
@@ -135,6 +135,118 @@ const getJiraCloudActions: () => Record<string, IBpmnAction> = () => {
                 formData: {},
             },
         },
+        [JiraCloudAction.GET_EPIC_WORKLOGS]: {
+            id: JiraCloudAction.GET_EPIC_WORKLOGS,
+            credentialType: ActionCredentialType.ATLASSIAN,
+            label: translate('Process.Details.Modeler.Actions.JiraCloud.GetEpicWorklogs.Label'),
+            script: JiraCloudAction.GET_EPIC_WORKLOGS,
+            runner: Runner.DESKTOP_SCRIPT,
+            output: {
+                assignVariables: true,
+                outputMethods: {
+                    variableName: '${content.output[0]}',
+                },
+            },
+            form: {
+                schema: {
+                    type: 'object',
+                    properties: {
+                        input: {
+                            title: translate('Process.Details.Modeler.Actions.Common.Input'),
+                            type: 'object',
+                            properties: {
+                                epic: {
+                                    title: translate('Process.Details.Modeler.Actions.JiraCloud.GetEpicWorklogs.Epic'),
+                                    type: 'string',
+                                },
+                                groupByDay: {
+                                    title: translate('Process.Details.Modeler.Actions.JiraCloud.GetEpicWorklogs.GroupByDay'),
+                                    type: 'boolean',
+                                },
+                                mode: {
+                                    title: translate('Process.Details.Modeler.Actions.JiraCloud.GetEpicWorklogs.DateMode'),
+                                    type: 'string',
+                                    enum: [dateMode.date.value, dateMode.period.value, dateMode.collection.value],
```

Comment:
I'd check if attendance form works with it - I've got error in similar case

Reply:
If there's no problem with executing attended process (no form validation error) you're good to go